nixos/modules/services/mail/schleuder.nix

```
@@ -0,0 +1,100 @@
+{ config, pkgs, lib, ... }:
+let
+  cfg = config.services.schleuder;
+  settingsFormat = pkgs.formats.yaml { };
+  postfixMap = entries: lib.concatStringsSep "\n" (lib.mapAttrsToList (name: value: "${name} ${value}") entries);
+  writePostfixMap = name: entries: pkgs.writeText name (postfixMap entries);
+in
+{
+  options.services.schleuder = {
+    enable = lib.mkEnableOption "Schleuder secure remailer";
+    lists = lib.mkOption {
+      description = "Addresses of lists that should be handled by Schleuder.";
```

```suggestion
      description = "List of list addresses that should be handled by Schleuder.";
```

This is otherwise easily misunderstood.